test/js_interop_helpers_test/shared_tests.dart

```diff
@@ -4,6 +4,8 @@ library js_function_test;
 import 'dart:html';
 
 import 'package:js/js.dart';
+import 'package:js/js_util.dart' show newObject;
+// TODO: remove in 5.0.0
```

Comment:
What needs to be removed?

Reply:
we no longer use the get/setProperty functions from `js_interop_helpers.dart` and `_dart_helpers.js` in favor of using `dart:js_util` instead, so we no longer need the test coverage.